### DISCUSSION TITLE
Use of Secret in setCookies

### DISCUSSION THREAD

**Discussion by noe-charmet** [codeceptjs/CodeceptJS#3558] [Ideas]
As cookies can contain session information, they could be considered as secrets. It would be great to be able to use a Secret there. To ensure we are not logging any sensitive data.

**Comment by kobenguyent** [comment]
Could you please provide your showcase in the real project?

**Comment by noe-charmet** [comment]
Currently we would be doing

```typescript
I.setCookie({name: 'authentication_cookie', value: 'someSuperSecret'})
```

Which would print the content of the cookie in the logs.

I would like to do 

```typescript
I.setCookie({name: 'authentication_cookie', value: secret('someSuperSecret')})
```

As to have the secret redacted in the logs
